sustainability_employee_commuting/tests/test_hr_employee.py

```diff
@@ -0,0 +1,57 @@
+from odoo.tests.common import TransactionCase
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class EmployeeCommutingTestCase(TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(EmployeeCommutingTestCase, cls).setUpClass()
```

Comment:
```suggestion
        super().setUpClass()
```

We prefer super like this